website/docs/reference/resource-properties/constraints.md

```diff
@@ -350,7 +350,62 @@ models:
 
 </File>
 
-Expected DDL to enforce constraints:
+### Column-level constraint on nested column:
+
+<File name='models/nested_column_constraints_example.sql'>
+
+```sql
+{{
+  config(
+    materialized = "table"
+  )
+}}
+
+select
+  'string' as a,
+  struct(
+    1 as id,
+    'name' as name,
+    struct(2 as id, struct('test' as again, '2' as even_more) as another) as double_nested
+  ) as b
+```
+
+</File>
+
+<File name='models/nested_fields.yml'>
+
+```yml
+version: 2
+
+models:
+  - name: double_nested_fields
+    config:
+      contract: {enforced: true}
+      materialized: table
```

Comment:
nit: don't think this is necessary since we have `config(materialized='table')` in the sql contents already